src/database/helpers.tsx

```diff
@@ -13,3 +13,15 @@ export const mapToObject = (map: Map<any, any>): Object => {
     ),
   );
 };
+
+export const firestoreAutoId = (): string => {
```

Comment:
Curious - what is this doing? How is this different from the autogenerated user id?

Reply:
it's weird - because we have id as an actual _field_ ALONGSIDE the id, firestore won't auto fill that field with the id it autogenerates, so we have to here, it's the same code that firebase uses to actually generate the uid i believe